### ISSUE TITLE
Collaborate?

### ISSUE THREAD

**Issue by Korijn** [panxinmiao/three-py#2]
Hi, I just saw your awesome project and wanted to check if you are aware of https://github.com/pygfx/pygfx ?

Seems like we are doing almost exactly the same things. Maybe we can help each other out?

As an example, we don't have support for PBR/standard material and lights yet, which your library seems to have implemented already. On the other hand, we already have other types like points, lines, images and volumes working. We can maybe benefit by working together?

**Comment by panxinmiao** [comment]
Thank you.

I am not a professional computer graphics developer. Most of the logic and algorithms in this project come from ThreeJs, including the "node material" system.

I started this project because I was doing some 3D computer vision work. My colleagues and I need a 3D geometry Library in Python ecology. We can use it ourselves and customize it freely to do some research and testing. So, I started this project while learning relevant knowledge.

Later, I found it might be useful to others, so I cleaned up the code and made it public on GitHub.

So, I'm afraid I can't help you a lot. But I will provide some help within my ability.

If you are interested in this project, I'm glad we can improve it together.
